packages/pages/src/dev/server/middleware/constants.ts

```diff
@@ -1,12 +1,12 @@
-export const viteDevServerPort = 5173;
+export const defaultDevServerPort = 5173;
```

Comment:
Would it be simpler to set devServerPort as a constant here like `export const devServerPort = getPort(`? Then we wouldn't have to pass it as a param to multiple places